pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -1613,6 +1614,22 @@
   csf_page_faq_answer_05: "Each CSF course includes 13-17 lessons designed for 45-minute periods. We recommend all students move from lesson to lesson at a pace set by the teacher. There are many teacher-led project levels designed to be experienced in unison while the skill-building lessons can be completed by students at their own pace."
   csf_page_faq_question_06: "What materials do I need for CS Fundamentals?"
   csf_page_faq_answer_06: "Many lessons have handouts that guide students through activities. These resources can be printed or assigned digitally. Some lessons call for typical classroom supplies and manipulatives. [Visit the CSF Syllabus to learn more](%{syllabus_url})."
+  csf_page:
+    additional_resources:
+      heading: "Additional resources"
+      desc: "Beyond curriculum and professional learning, we have many materials to support your classroom."
+      catalog:
+        title: "Curriculum Catalog"
+        desc: "Comprehensive curriculum offerings for every grade and experience level featuring robust structured and self-paced learning options."
```

Comment:
Nit: should there be commas between the adjectives at the end so it says "featuring robust, structured, and self-paced learning options" ?